node/pkg/watchers/solana/client.go

```diff
@@ -598,14 +599,20 @@ OUTER:
 
```

Comment:
Change looks good but I do think we might be able to remove the above RPC call to get inners:
https://github.com/gagliardetto/solana-go/blob/main/rpc/getTransaction.go#L40-L94

Reply:
I made this change.